fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/mixin/object/builder/AbstractBlockSettingsAccessor.java

```diff
@@ -76,6 +77,9 @@ public interface AbstractBlockSettingsAccessor {
 	float getVelocityMultiplier();
 
 	@Deprecated
+	@Accessor
+	float getJumpVelocityMultiplier();
+
 	@Accessor
```

Comment:
The above accessor stole this one's `@Deprecated` mark
```suggestion
	@Deprecated
	@Accessor
```